services/core/methods/controllers/accounts.js

```diff
@@ -31,22 +32,27 @@ const getDataForAccounts = async params => {
 	response.meta = {};
 	response.links = {};
 
+	const accountDataCopy = JSON.parse(JSON.stringify(accounts.data));
```

Comment:
Do we need to clone the response? 
This would fail if `accounts.data` contained any `BigInt` values. `JSON.stringify` doesn't handle them by default.

Reply:
It should map them to `string` shouldn't it?

Reply:
Without it rank is always `NaN`, for some reason these properties are affected by passing a reference.

Reply:
`JSON.stringify` doesn't automatically parse it to a string. We could utilize the `parseToJSONCompatObj` utility from `services/core/shared/core/compat/common/utils.js` in `development` branch.